typing/parmatch.ml

```diff
@@ -2438,39 +2438,38 @@ let pattern_stable_vars ns p =
   let module M_mod = unpack M .. in true
 *)
 
+open Tast_iterator
```

Comment:
A local open would probably work as well. We have to have an open statement somewhere otherwise the compiler will complain about fields that aren't visible in the current scope.

Reply:
You can use `record.Module.field` to access a field declared in another record. But in that case, opening just within the scope where the iterator is used would also be fine.

Reply:
Another option would be to use a short alias for `Tast_iterator` something like:
```OCaml
module Iter = Tast_iterator
```

Reply:
Yes, but still, not in the global scope, when only one sub-part of one function uses the iterator.